fedimint-client/src/lib.rs

```diff
@@ -1860,6 +1874,55 @@ impl Client {
                 )
             })
     }
+
+    /// Blocks till the client has synced the guardian public key set
+    /// (introduced in version 0.4) and returns it. Once it has been fetched
+    /// once this function is guaranteed to return immediately.
+    pub async fn get_guardian_public_keys_blocking(
+        &self,
+    ) -> BTreeMap<PeerId, secp256k1_zkp::PublicKey> {
+        self.db.autocommit(|dbtx, _| Box::pin(async move {
+            let config = self.config().await;
+
+            let guardian_pub_keys = if let Some(guardian_pub_keys) = config.global.broadcast_public_keys {guardian_pub_keys}else{
```

Comment:
formatting oddly off

Reply:
idk what it is, `just format` doesn't do anything, I'd say that's an unrelated tooling/CI issue we should track independently